src/renderer/value-transformer/KotlinValueTransformer.ts

```diff
@@ -16,7 +16,7 @@ import { ValueTransformer } from './ValueTransformer';
 export class KotlinValueTransformer implements ValueTransformer {
   constructor(private readonly typeNameMap: Record<string, string>) {}
 
-  convertValueType(valueType: ValueType): string {
+  convertValueType(valueType: ValueType, uniquePath: string): string {
```

Comment:
Can you remove all those `uniquePath` and `ownerName`? I suppose they are no longer needed.

Reply:
The same goes for `serializers.ts`.